docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -760,3 +760,32 @@ Then `${info.duration}` is > `1000`
 Then `${info.networkState}` is = `2`
 Then `${info.src}` matches `.+youtube.+`
 ----
+
+=== Perform steps for all elements
+
+Executes the steps against all elements found by locator.
```

Comment:
we need to define explicitly, that the current iteration element is set as a search context, and describe how context is managed by the step